share/spice/mars_photos/footer.handlebars

```diff
@@ -0,0 +1,2 @@
+<div>Rover: {{this.rover.name}}</div>
+<div>Camera: {{ellipsis this.camera.full_name}}</div>
```

Comment:
@jdorweiler seeing as we're just using two things here, we can set these in the JS as `title` and `subtitle` and ditch the footer altogether